pkg/probe/http/http.go

```diff
@@ -113,8 +115,11 @@ func DoHTTPProbe(req *http.Request, client GetHTTPInterface) (probe.Result, stri
 			klog.V(4).Infof("Probe terminated redirects for %s, Response: %v", url.String(), *res)
 			return probe.Warning, fmt.Sprintf("Probe terminated redirects, Response body: %v", body), nil
 		}
+		if limitReached {
+			return probe.Warning, fmt.Sprintf("Non fatal body truncation for %s, Response: %v", url.String(), body), nil
```

Comment:
We are outputting 10kb size message (body of response) to user in `probe.Warning` and these messages are not able to help user troubleshooting, right?
So can we avoid outputting the noise by not outputting the body of response to the message?

Reply:
indeed, thought the same, left it in for consistency reasons as the other returns even though it can be 10 kb are still outputting it, shall i change it in this particular case?